src/event.mli

```diff
@@ -0,0 +1,9 @@
+(* event.mli *)
+type t =
+  | KeyDown of Key.t
+  | KeyUp of Key.t
+  | MouseButtonDown of Mouse.button * (int * int)
+  | MouseButtonUp of Mouse.button * (int * int)
+  | MouseMotion of (int * int)
+  | MouseWheel of int
+  | MouseDrag of Mouse.button * (int * int)
```

Comment:
Can you just add an odoc comment or two to this header file explaining what it represents?